### PR TITLE
add a file for wsp mn

### PR DESCRIPTION
**Pull request by john-schneider**
The purpose of this PR is to add a hometown file for WSP MN.

Some ? that I want you all to help m answer:
1. If you've been to Orlando, what did you do?  What did you think?
1. If you have kids, what kind of things do they like to do at night?
1. What kind of food do you like?

### CODE REVIEW

**Comment by chris-kerber** [comment]
I like pizza and burgers

File: john-schneider-wsp.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# West St Paul, MN
+
+### Restaurants
+
+### Fast Food
+Burger Time
+
+### One of a kind
+5 - 8 Club
```
Comment (chris-kerber) [inline john-schneider-wsp.md:18]: Love the juicy lucy

**Comment by chris-kerber** [comment]
> If you have kids, what kind of things do they like to do at night?

No kids yet

**Comment by john-schneider** [comment]
this pull request resolves #19

**Review by chris-kerber — APPROVED** [review]
Looks good.  Ready to :ship:

**Pull request merged**